.gitignore

```diff
@@ -58,6 +58,7 @@ anaconda-mode/
 *.dll
 *.so
 *.dylib
+bin/
```

Comment:
Line 4 has a `bin/manager` entry. Maybe merge these two to avoid duplicated entries.